### PR TITLE
Docs update on running docker container with docker desktop

### PR DESCRIPTION
**Pull request by edan-bainglass**
- Update language
- Add section describing how to spin up an aiidalab/qe container using Windows Docker Desktop

### CODE REVIEW

File: docs/source/installation/access_aiidalab/container.rst [hunk 1]
```diff
@@ -1,43 +1,77 @@
...
-   If you already have access to AiiDAlab but the Quantum ESPRESSO app is not installed, you need proceed to :doc:`install the app </installation/install>`.
+   If you used another method to access AiiDAlab, you may proceed to :doc:`install the app </installation/install>`.
 
 AiiDAlab is available as a Docker container - a self-contained, pre-configured environment including all the necessary software to access the AiiDAlab platform.
-We also provide container that includes the Quantum ESPRESSO app installed and ready to use.
-To run the container, you first need to install Docker on your local machine.
-If you have yet to do so, you may click `here <https://docs.docker.com/get-docker>`_ to follow the official Docker installation guide.
-
-We recommended to install the `Docker Desktop <https://docs.docker.com/desktop/>`_ which is available for all major operating systems.
+Conveniently, we provide a bluprint (image) for such a container with the Quantum ESPRESSO app pre-installed and ready for use. To run the container, you first need to `install Docker <https://docs.docker.com/get-docker>`_ on your local machine.
```
Comment (unkcpz) [inline docs/source/installation/access_aiidalab/container.rst:10]:
```suggestion
Conveniently, we provide a blueprint (image) for such a container with the Quantum ESPRESSO app pre-installed and ready for use. To run the container, you first need to `install Docker <https://docs.docker.com/get-docker>`_ on your local machine.
```

File: docs/source/installation/access_aiidalab/container.rst [hunk 2]
```diff
@@ -1,43 +1,77 @@
...
 
 .. important::
 
-   The following steps require a local installation of Docker.
-   You can check the docker is properly installed by running ``docker run hello-world`` in the terminal.
+   The following steps require a local installation of Docker. You can verify your Docker installation by running ``docker run hello-world`` in the terminal.
 
-`AiiDAlab launch`_ is a thin Docker wrapper which takes care of all the prerequisites to run the AiiDAlab Docker image.
-It helps to manage multiple AiiDAlab profiles, each with its own home directory for persistent storage, and allows to easily switch between them.
+`AiiDAlab launch`_ is a thin Docker wrapper which takes care of all the prerequisites to run the AiiDAlab Docker image. It helps to manage multiple AiiDAlab profiles, each with its own home directory for persistent storage, and allows to easily switch between them.
```
Comment (unkcpz) [inline docs/source/installation/access_aiidalab/container.rst:74]:
```suggestion
`AiiDAlab launch`_ is a thin Docker wrapper which takes care of all the prerequisites to run the AiiDAlab Docker image. 
It helps to manage multiple AiiDAlab profiles, each with its own home directory for persistent storage, and allows to easily switch between them.
```

File: docs/source/installation/access_aiidalab/container.rst [hunk 3]
```diff
@@ -1,43 +1,77 @@
...
+   .. note::
+
+      Subsequent connections to the port in the browser will not prompt for a token for some time. If and when it does, you may again retrieve the token from the log.
+
+.. important::
+
+   To avoid losing your work when the container shuts down (manually, or when the machine is turned off), it is important to associate the container with a volume - a local directory - with which the container data's is mirrored. When set up, the container will restart from this mirrored volume.
 
-Comming soon with video tutorial.
+Note that Docker Desktop is also available for MacOS and Linux. However, you may prefer to use the command line interface (CLI). If so, proceed to the following sections for instructions.
```
Comment (unkcpz) [inline docs/source/installation/access_aiidalab/container.rst:65]:
```suggestion
Note that Docker Desktop is also available for MacOS and Linux. 
However, you may prefer to use the command line interface (CLI). 
If so, proceed to the following sections for instructions.
```

File: docs/source/installation/access_aiidalab/container.rst [hunk 4]
```diff
@@ -1,43 +1,77 @@
...
+   The Docker installation link above walks you through installing Docker Desktop - a convenient graphical user interface to Docker. However, if you have chosen instead to install the `docker engine <https://docs.docker.com/engine/install/ubuntu/>`_ directly for use via a terminal, if **(and only if)** you are on a Linux system, you will need `root` privileges to perform the `post-installation steps for Docker Engine <https://docs.docker.com/engine/install/linux-postinstall/>`_.
 
-Once Docker is installed, you can launch the container in one of several ways depending on your operating system.
-This is discussed in the following sections.
+Once Docker is installed, you can launch the container in one of several ways depending on your operating system. This is discussed in the following sections.
```
Comment (unkcpz) [inline docs/source/installation/access_aiidalab/container.rst:16]:
```suggestion
Once Docker is installed, you can launch the container in one of several ways depending on your operating system. 
This is discussed in the following sections.
```

File: docs/source/installation/access_aiidalab/container.rst [hunk 5]
```diff
@@ -1,43 +1,77 @@
...
 
 Docker Desktop
 **************
 
-If you use Windows, this may be the easiest way to launch the container.
-For MacOS and Linux, you can also use Docker Desktop, but you may prefer to use the command line interface (CLI), see the next sections.
+If you use Windows, you may choose to use the Docker Desktop interface to launch the container. Once Docker Desktop is installed (see above link), follow these instructions to spin up an AiiDAlab container.
```
Comment (unkcpz) [inline docs/source/installation/access_aiidalab/container.rst:21]:
```suggestion
If you use Windows, you may choose to use the Docker Desktop interface to launch the container. 
Once Docker Desktop is installed (see above link), follow these instructions to spin up an AiiDAlab container.
```

**Review by unkcpz — COMMENTED** [review]
Thanks @edan-bainglass, it is much more clear. I added a commit to break lines to keep one line for one sentence, which is the convention we used in both `aiida-core` doc and here.

**Review by unkcpz — APPROVED** [review]

**Comment by edan-bainglass** [comment]
Thanks @unkcpz. Even caught a typo 😅
All good with one sentence per line rule. Will keep that in mind for next time.

**Pull request merged**